deploy/rbac-permissions-operator-config/05-osd-readers.ClusterRole.yaml

```diff
@@ -39,51 +39,377 @@ rules:
   verbs:
   - get
   - list
-### START - view ClusterVersion
+### START - run `oc adm top pod` or `oc adm top node`
 - apiGroups:
-  - config.openshift.io
+  - metrics.k8s.io
   resources:
-  - clusterversions
+  - nodes
+  - pods
   verbs:
   - list
+### END
+### Start - fix `oc describe nodes`
+# https://issues.redhat.com/browse/OHSS-1105
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
   - get
-  - watch
 ### END
-### START - run `oc adm top pod` or `oc adm top node`
+### Start - Allow read permissions to more resources
+# https://issues.redhat.com/browse/OSD-5537
 - apiGroups:
-  - metrics.k8s.io
+  - admissionregistration.k8s.io
   resources:
-  - nodes
-  - pods
+  - '*'
   verbs:
+  - get
   - list
-### END
-### Start - Allow viewing platform and other infra data in console
-# https://issues.redhat.com/browse/OSD-4298
+  - watch
 - apiGroups:
-  - config.openshift.io
+  - apiextensions.k8s.io
   resources:
-  - infrastructures
```

Comment:
Why is infrastructures removed?  Please add it back.

Reply:
Clusterversions will be included in line 162:

```
# access resources like oauth, clusterversion,clusteroperators
- apiGroups:
  - config.openshift.io
  resources:
  - oauths
  - '*'
  verbs:
  - get
  - list
  - watch
```